plugins/CoreHome/stylesheets/dataTable/_dataTable.less

```
@@ -630,9 +641,10 @@ td.cellSubDataTable .loadingPiwik {
     padding-left:8px;
   }
 }
-
-div.dataTableScroller {
-  overflow-x: scroll;
+.widgetContent,.reportContainer {
```

This actually prevents the sticky header from working in the goals reports, even if there would be enough space.
It's actually not needed anymore as the javascript will add the scrollbar if needed.
```suggestion
.widgetContent {
```